string LEFT = LEFT
string RIGHT = RIGHT
string FRONT = FRONT
string CLEAR = CLEAR
string OBSTACLE_ON
